### PR TITLE
Add jest tests of frontend to CI

### LINKED ISSUE #223: Add jest tests of frontend to CI

### PR DESCRIPTION
**Pull request by wrc9817**

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]

File: src/front-end/src/App.test.js [hunk 1]
```diff
@@ -1,9 +1,3 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
 test('renders learn react link', () => {
```
Comment (jpabadir) [inline src/front-end/src/App.test.js]: Minor: can you rename this to "sample test" or something like that?

**Review by jpabadir — APPROVED** [review]
This is great! It's awesome that you managed to make CI run the unit tests. Great work! 

Please make sure you address my minor comment before merging.

File: src/front-end/src/App.test.js (hunk 1 above)
Comment (wrc9817) [reply src/front-end/src/App.test.js]: I just did!

Comment (wrc9817) [reply src/front-end/src/App.test.js]: Thanks for mentioning that

**Review by katmartel — APPROVED** [review]
Great work!

**Pull request merged**